js/blocks/loader/edit.js

```diff
@@ -12,14 +12,16 @@ const formControls = ( props, block ) => {
 	const fields = simplifiedFields( block.fields ).map( field => {
 
 		// If its not meant for the inspector then continue (return null).
-		// if ( !field.location ) {
 		if ( !field.location || !field.location.includes( 'editor' ) ) {
 			return null
 		}
 
+		const controlFunction = controls[ field.control ];
```

Comment:
So this line is saying that whatever my control is called, there needs to be a matching JS function?

Reply:
@lukecarbis correct... but it doesn't allow for extensibility in the way you might think. Lets sync.